### PR TITLE
Fix #628 (and undefined error when starting a currency)

### LINKED ISSUE #628: Cli: gen-root error (missing promise)
```
? Newcomers to add: (Press <space> to select)
❯◯ BenoitLavenier
 ◯ LudovicPecquot 2016-09-30T10:00:33+02:00 - error:  httpCode=400, ucode=2018, message=Wrong new block: cannot make a root block without members
2016-09-30T10:00:33+02:00 - error:  Error: [object Object]
    at Error (native)
    at /home/blavenie/git/blavenie/duniter/app/cli.js:828:15
    at process._tickCallback (node.js:369:9)
2016-09-30T10:00:33+02:00 - debug: Trying to close SQLite...
2016-09-30T10:00:33+02:00 - error:  Error: [object Object]
    at Error (native)
    at /home/blavenie/git/blavenie/duniter/app/cli.js:828:15
    at process._tickCallback (node.js:369:9)

```

### PR DESCRIPTION
**Pull request by blavenie**
WARN : `install.sh` changed should be ignored (don't now how to do that in a pull request)

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -15,7 +15,7 @@ latest_version() {
 }
 
 repo_url() {
-  echo "https://github.com/duniter/duniter.git"
+  echo "https://github.com/blavenie/duniter.git"
```
Comment (M5oul) [inline install.sh]: ?

Comment (blavenie) [inline install.sh]: oui, cf mon message (WARN) au dessus

Comment (M5oul) [inline install.sh]: You could:
- add a new commit to revert that commit.
- `drop` that commit using [`git rebase`](https://stackoverflow.com/questions/1186535/how-to-modify-a-specified-commit-in-git).

Modifications done on your branch will automatically be updated here.

**Comment by blavenie** [comment]
Je rouvre un autre PR plus clean

**Pull request closed without merging**